.travis.yml

```diff
@@ -143,6 +143,8 @@ before_install:
           fi
         - export PKG_CONFIG_PATH="$HOME/cunit-install/$CROSS_ARCH/lib/pkgconfig:${PKG_CONFIG_PATH}"
         - find $HOME/cunit-install
+          # workaround for tap driver issues on recent Travis images
+        - sed -e '/tap/d' -i platform/linux-generic/test/Makefile.am
```

Comment:
@lumag take iptables line from here to make tap work https://github.com/Linaro/odp/pull/615/commits/3f0d341faad21b57e3d980f3df5b0b54701c7fbd

Reply:
@muvarov thanks for the suggestion